redaxo/src/addons/mediapool/functions/function_rex_mediapool.php

```diff
@@ -324,7 +324,9 @@ function rex_mediapool_syncFile($physical_filename, $category_id, $title, $files
 /**
  * @param string $filename
  *
- * @return bool
+ * @return (bool|string)[]
```

Comment:
für php storm invalid
```suggestion
 * @return bool[]|string[]
```